src/Uno.UI.Runtime.Skia.Gtk/Extensions/Storage/Pickers/FileSavePickerExtension.cs

```diff
@@ -41,7 +42,11 @@ public FileSavePickerExtension(FileSavePicker owner)
 			StorageFile? file = null;
 			if (dialog.Run() == (int)ResponseType.Accept)
 			{
-				file = await StorageFile.GetFileFromPathAsync(dialog.Filename);
+				if (!File.Exists(dialog.Filename))
+				{
+					File.WriteAllText(dialog.Filename, "");
```

Comment:
This may create a file with a BOM depending on the default encoding, wouldn't it ?

Maybe use `File.WriteAllBytes(dialog.Filename, Array.Empty<byte>());` or `File.Create(dialog.Filename).Dispose();` instead.